components/SubmissionForm.vue

```diff
@@ -27,17 +27,31 @@
             type="text"
             required
             class="mb-5 px-3 py-3.5 w-[350px] h-[50px] bg-white rounded-lg border border-zinc-400
-          text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
+            text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
             :placeholder="$t('submitPage.location')"
+            @blur="initialValidationCheck(submissionStore.location, 'googleMaps')"
         >
         <span
             class="mb-2 text-neutral-600 text-sm font-normal font-['Noto Sans JP']"
         >{{ $t('submitPage.healthcareProfessionalName') }}</span>
-        <p
-            v-show="!isValidLastName || !isValidFirstName"
-            class="text-currentColor text-xs font-['Noto Sans JP']"
-        >
-            {{ $t('submitPage.lastNameValidation') }}
+        <p class="flex text-currentColor text-xs font-['Noto Sans JP']">
+            <span
+                :class="[
+                    !isValidLastName ? 'visible' : 'invisible',
```

Comment:
why use `visible` vs `v-show`?

Reply:
I use this because v-show uses hidden which does not keep the spacing. I wanted to keep the spacing so the span for the first name would appear above the First name input.

If I use v-show and the last name is valid but the first name isn't the error will appear above the last name.